src/orion/client/experiment.py

```diff
@@ -693,8 +693,9 @@ def workon(
         """
         self._check_if_executable()
 
+        # TODO: should this n_workers always be smaller than executor configured n_workers?
```

Comment:
Maybe issue a warning if n_workers is greater than the n_workers of executor. That's certainly a scenario we would like to avoid, but maybe it's not critical enough to raise an error.

Reply:
yes, the only issue when n_workers is bigger than executor configured, at the end it will try to run the experiment at free workers and find the experiment already done.